eng/common/testproxy/test-proxy-tool.yml

```diff
@@ -25,13 +22,13 @@ steps:
   - pwsh: |
       Start-Process $(Build.BinariesDirectory)/test-proxy/test-proxy.exe `
         -ArgumentList "--storage-location '${{ parameters.rootFolder }}'" `
-        -NoNewWindow -PassThru
+        -NoNewWindow -PassThru -RedirectStandardOutput $(Build.SourcesDirectory)/test-proxy.log
     displayName: 'Run the testproxy - windows'
     condition: and(succeeded(), eq(variables['Agent.OS'],'Windows_NT'))
 
   # nohup does NOT continue beyond the current session if you use it within powershell
   - bash: |
-      nohup $(Build.BinariesDirectory)/test-proxy/test-proxy &
+      nohup $(Build.BinariesDirectory)/test-proxy/test-proxy > $(Build.SourcesDirectory)/test-proxy.log &
```

Comment:
@HarshaNalluru instead of `docker logs ambition_azsdk_test_proxy` we will `cat $(Build.SourcesDirectory/test-proxy.log`.

(or `more` if you use bash instead of `pwsh` )

Reply:
So, what is it that I or the other languages need to do in our CIs/jobs?

Reply:
Exactly what I listed above. `cat` is powershell for "dump file contents", `more` is the bash equivalent.

Reply:
And the target file contents (when `test-proxy-tool.yml` is invoked) will be under `$(Build.SourcesDirectory/test-proxy.log`.

Reply:
@scbedd FYI `cat` is bash, powershell just aliased their own command to cat because they did that for all common GNU utilities. `more` paginates output and requires key input to scroll, so you'd want to use `cat` for both pwsh and bash.

Reply:
Also, do you print any stderr from test proxy? I don't think this will capture it, you'd need to do `&>` instead of `>`.

Reply:
100%. Do you notice the commits where I was _trying_ to enable that? 😠

Reply:
```suggestion
      nohup $(Build.BinariesDirectory)/test-proxy/test-proxy &> $(Build.SourcesDirectory)/test-proxy.log &
```

Reply:
That didn't actually work. :( Reverted. I'll tackle this in a follow-up PR.